test/schema.rb

```diff
@@ -8,8 +8,8 @@
 
   # Add a table only the shard database will have. Conditional
   # exists since Phenix loads the schema file for every database.
-  if ActiveRecord::Base.connection.current_database == 'arhp_test_1_shard'
-    create_table 'test1_shards' do |t|
+  if ActiveRecord::Base.connection.current_database == 'arhp_test_db_c'
+    create_table 'pool1_db_cs' do |t|
```

Comment:
Review tip: The `cs` here is due to us mimicking the pluralization functionality of Rails. E.g. shard model -> shards table